sdk/go/auto/workspace.go

```diff
@@ -183,11 +183,18 @@ type StackSummary struct {
 	URL              string `json:"url,omitempty"`
 }
 
+// TokenInformation contains information about the currently logged-in Pulumi access token.
+type TokenInformation struct {
+	Name         string `json:"name"`
+	Organization string `json:"organization"`
+}
```

Comment:
In general I think we'll want this (and other SDK types) to mimic `TokenInformation` as described internally, which is in `sdk/go/common/workspace/creds.go` (as referenced from the `pulumi whoami` command in `pkg/cmd/pulumi/whoami.go`'s `WhoAmIJSON` type). That means that `Organization` is optional and should be `omitempty`, and we should probably add the `Team` field (also optional) while we're here.

Reply:
Done.